.github/workflows/full-build.yml

```diff
@@ -6,7 +6,7 @@ on:
     types: [published, prereleased]
 env:
   python_version: '3.10'
-  mainline_build: ${{ github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/heads/packaging/') }}
+  mainline_build: ${{ github.ref == 'refs/heads/main' }} || ${{ github.event.label.name == 'translation' }}
```

Comment:
Are the extra curly braces really needed?

Reply:
Thanks, fixed!